onnxruntime/core/providers/cuda/rnn/cudnn_rnn_base.h

```diff
@@ -42,7 +42,7 @@ class CudnnRNN {
     if (!cudnn_rnn_desc_)
       CUDNN_RETURN_IF_ERROR(cudnnCreateRNNDescriptor(&cudnn_rnn_desc_));
 
-    CUDNN_RETURN_IF_ERROR(cudnnSetRNNDescriptor(cudnnHandle,
+    CUDNN_RETURN_IF_ERROR(cudnnSetRNNDescriptor_v6(cudnnHandle,
```

Comment:
you can't change this just yet.
the api is changed for cuDNN 8 but we still need to use cuDNN 7.x for production